### ISSUE TITLE
MiKo_2223 should consider parameters as well

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#1138]
Currently, MiKo_2223 does not detect
```c#
/// You may want to call stream.Seek() before.
```
in case `stream` is a parameter.

It should also detect and report such invocations.
